README.md

```diff
@@ -18,22 +18,46 @@ pip install git+https://github.com/Imageomics/sum-buddy
 ### Command Line Usage
 
 ```
-usage: sum-buddy [-h] --input-dir INPUT_DIR --output-file OUTPUT_FILE
+usage: sum-buddy [-h] --input-dir INPUT_DIR --output-file OUTPUT_FILE [--ignore-file IGNORE_FILE] [--include-file INCLUDE_FILE] [--algorithm ALGORITHM]
 
-Generate CSV with filepath, filename, and MD5 checksums for all files in a given directory
+Generate CSV with filepath, filename, and checksums for all files in a given directory
 
 options:
-  -h, --help                         show this help message and exit
-  --input-dir INPUT_DIR              Directory to traverse for files
-  --output-file OUTPUT_FILE          Filepath for the output CSV file
+  -h, --help            show this help message and exit
+  --input-dir INPUT_DIR
+                        Directory to traverse for files
+  --output-file OUTPUT_FILE
+                        Filepath for the output CSV file
+  --ignore-file IGNORE_FILE
+                        Filepath for the ignore patterns file
+  --include-file INCLUDE_FILE
+                        Filepath for the include patterns file
+  --algorithm ALGORITHM
+                        Hash algorithm to use (default: md5)
 ```
 
```

Comment:
```suggestion
Note: this package is still under development and currently only supports `md5` checksums.

```
We'll probably want to provide a list of options as we add them, but this is probably sufficient for v0.1.0.

Reply:
Actually it already works with everything available through `hashlib`. The list can be seen with `hashlib.algorithms_available`.

Reply:
Ah, fabulous! See updated suggestion above (also added to help text) :)